### ISSUE TITLE
Cancel button is not closing the color-picker

### ISSUE THREAD

**Issue by directcuteo** [zefoy/ngx-color-picker#209]
I have a simple problem for which I can't find a solution. I have the following code: 
`

           <div 
             [(colorPicker)]="selectedColor"
             [style.background]="selectedColor"
             [cpToggle]="showColorPicker"
             [cpDialogDisplay]="'inline'"
             [cpCancelButton]="true"
             [cpCancelButtonClass]="'btn btn-light color-picker-cancel'"
             [cpCloseClickOutside]="false"
             [cpSaveClickOutside]="false"
             [cpOKButton]="true"
             [cpOKButtonText]="'Choose'"
             [cpOKButtonClass]="'btn btn-success'"
             [cpWidth]="'190px'"
             [cpHeight]="'265px'"
             (colorPickerSelect)="onColorSelect()"
             (colorPickerChange)="onColorSelect($event)"></div>
``
I am showing the color picker using `[cpToggle]="showColorPicker"` and this value is changing from an external button (should not be a problem). The problem that I have is that the 'Cancel' button is not closing the color picker (like in the StackBlitz example does). I would like to be able to close it from that button because it will undo to the selected color before the color picker was displayed.
I tried something like: `(colorPickerCancel)="showColorPicker = !showColorPicker"` but didn't work either.

**Comment by sconix** [comment]
You have set the display type to the inline, so its not a popup so cancel button won't close dialog since there is none.

**Comment by directcuteo** [comment]
Yes, tested on StackBlitz and with 'inline' value won't work. But in my example it still doesn't work at all anyway. I've added the cancel button to the StackBlitz example with the Toggle Button which looks like what I want and it is working. I have copied whole HTML code to my app and that Cancel button again doesn't work.

I provide here a simple example like the one with Toggle button from StackBlitz but where the cancel button is not working for me.

`@Component({`
`selector: 'app-cancel-color-picker',`
`template: `
`'
    <button class="btn btn-primary" (click)="showColorPicker = !showColorPicker;" #ignoredButton> 
    Toggle </button>`
`      <input #ignoredInput [style.background]="selectedColor" [cpFallbackColor]="'#f200bd'" 
       [cpIgnoredElements]="[ignoredButton, ignoredInput]" [(cpToggle)]="showColorPicker" 
       [(colorPicker)]="selectedColor" [cpCancelButton]="true"/> 
'`
`})`
` export class CancelColorPickerComponent implements OnInit {`

`showColorPicker = false;`
`selectedColor = '#ff0000';
`
`constructor(public vcRef: ViewContainerRef, private cpService: ColorPickerService) { }`

`ngOnInit() { }`
`
} `
